packages/expo-cli/src/commands/init.ts

```diff
@@ -161,53 +163,194 @@ async function action(projectDir: string, command: Command) {
+      );
+      log.nested('');
+      log.nested(`  cd ${cdPath || '.'}/ios`);
+      log.nested(`  pod install`);
+      log.nested('');
+    }
+  } else {
+    log.newLine();
+    logProjectReady({ cdPath, packageManager, workflow: 'managed' });
+  }
+}
+
+function logProjectReady({
+  cdPath,
+  packageManager,
+  workflow,
+}: {
+  cdPath: string;
+  packageManager: string;
+  workflow: 'managed' | 'bare';
+}) {
+  log.nested(chalk.bold(`✅ Your project is ready!`));
+  log.newLine();
+
+  // empty string if project was created in current directory
+  if (cdPath) {
     log.nested(
-      `Before running your app on iOS, make sure you have CocoaPods installed and initialize the project:`
+      `To run your project, navigate to the directory and run one of the following ${packageManager} commands.`
     );
-    log.nested('');
-    log.nested(`  cd ${cdPath || '.'}/ios`);
-    log.nested(`  pod install`);
-    log.nested('');
-    log.nested('Then you can run the project:');
-    log.nested('');
-    if (cdPath) {
-      // empty string if project was created in current directory
-      log.nested(`  cd ${cdPath}`);
-    }
-    log.nested(`  ${packageManager === 'npm' ? 'npm run android' : 'yarn android'}`);
-    log.nested(`  ${packageManager === 'npm' ? 'npm run ios' : 'yarn ios'}`);
+    log.newLine();
+    log.nested(`- ${chalk.bold('cd ' + cdPath)}`);
   } else {
-    log.nested(`To get started, you can type:\n`);
-    if (cdPath) {
-      // empty string if project was created in current directory
-      log.nested(`  cd ${cdPath}`);
+    log.nested(`To run your project, run one of the following ${packageManager} commands.`);
+    log.newLine();
+  }
+
+  if (workflow === 'managed') {
+    log.nested(
+      `- ${chalk.bold(
+        `${packageManager} start`
+      )} # you can open iOS, Android, or web from here, or run them directly with the commands below.`
+    );
+  }
+  log.nested(`- ${chalk.bold(packageManager === 'npm' ? 'npm run android' : 'yarn android')}`);
+
+  let macOSComment = '';
+  const isMacOS = process.platform === 'darwin';
+  if (!isMacOS && workflow === 'bare') {
+    macOSComment = ' # you need to use macOS to build the iOS project';
+  } else if (!isMacOS && workflow === 'managed') {
+    macOSComment = ' # requires an iOS device or access to a macOS for a simulator';
+  }
+  log.nested(
+    `- ${chalk.bold(packageManager === 'npm' ? 'npm run ios' : 'yarn ios')}${macOSComment}`
+  );
+
+  log.nested(`- ${chalk.bold(packageManager === 'npm' ? 'npm run web' : 'yarn web')}`);
+
+  if (workflow === 'bare') {
+    log.newLine();
+    log.nested(
+      `💡 You can also open up the projects in the ${chalk.bold('ios')} and ${chalk.bold(
+        'android'
+      )} directories with their respective IDEs.`
+    );
+  }
+}
+
+async function installPodsAsync(projectRoot: string) {
+  let step = logNewSection('Installing CocoaPods.');
+  if (process.platform !== 'darwin') {
```

Comment:
This is the third time we calculate this static property, maybe a constant would make it more readable...